### PR TITLE
dapp listing: add sarafu network

### PR DESCRIPTION
**Pull request by kamikazechaser**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V1 or DAppKit;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

### CODE REVIEW

File: locales/base.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely",
     "gooddollar": "Claim UBI daily or stake to earn and support UBI",
     "jumptask": "Earn crypto as you play, solve or share",
-    "masa": "Claim a free .celo domain, refer friends and earn rewards"
+    "masa": "Claim a free .celo domain, refer friends and earn rewards",
+    "sarafu": "Kenya's leading Economic Commons platform"
```
Comment (kamikazechaser) [inline locales/base.json:53]: @williamluke4 @WillRuddick Thoughts on this single line description?

Comment (williamluke4) [reply locales/base.json:53]: (larger than just Kenya) ... please change to
```suggestion
    "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics"
```

**Comment by MuckT** [comment]
@kamikazechaser on sarafu.network the 'Read More' option goes to a technical document. Would it be possible to replace this with something less technical such as the [blog](https://grassecon.org/category/blog)?

**Comment by kamikazechaser** [comment]
Sure, we will get that updated.

**Comment by MuckT** [comment]
@kamikazechaser is this still being worked on?

**Comment by kamikazechaser** [comment]
Updated "Read More" -> https://docs.grassecon.org/

**Comment by kamikazechaser** [comment]
Let me know if we need to change anything else. I'll fix the conflicts.

**Comment by MuckT** [comment]
@kamikazechaser looks like a small formatting issue please run `yarn format` and commit the changes.

**Comment by kamikazechaser** [comment]
@MuckT Done!

**Comment by MuckT** [comment]
@kamikazechaser thanks for all your hard work. Glad to see this getting merged!

**Review by MuckT — APPROVED** [review]

**Comment by kamikazechaser** [comment]
Thanks a lot!

**Pull request merged**